### PR TITLE
fix(dev-server): isDepsExists add non pkgPath judege

### PR DESCRIPTION
**Pull request by fireairforce**
# PR Details

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/toolkit/utils/src/is/index.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import fs from 'fs';
 import path from 'path';
+import { createDebugger } from 'src/debug';
```
Comment (Amour1688) [inline packages/toolkit/utils/src/is/index.ts:3]: src ?

Comment (fireairforce) [reply packages/toolkit/utils/src/is/index.ts]: updated

**Comment by fireairforce** [comment]
image
这里应该不是我这个 changeset 导致的

**Comment by caohuilin** [comment]
> image 这里应该不是我这个 changeset 导致的

嗯 先忽略，我一会改一下校验脚本吧

**Pull request merged**